app/views/projects/_project_assignment_form.html.erb

```diff
@@ -1,8 +1,7 @@
 <%= form_with model: project, url: assign_project_path(project), method: :patch, class: 'form-inline' do |form| %>
```

Comment:
I still get sent down the original assign path in the projects controller by `assign_project_path(project)`

I wonder if it's my local db state. but if an application has no current assignments, Would we expect to back populate all production applications to have an application manager?